lib/api.ts

```diff
@@ -52,6 +52,35 @@ export class Api {
         }))
     }
 
+    async createBikeSharingInvitation(shareinfo: any): Promise<any> {
+        let req = await fetch(`/api/api_vanmoof_com/createBikeSharingInvitation`, {
+            method: 'POST',
+            headers: {
+                'Api-Key': API_KEY,
+                'Authorization': 'Bearer ' + this.credentials.token,
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(shareinfo)
+        })
+
+        if (req.status === 200) {
+            const res = await req.json()
+            // setSuccessModal(true) - TODO
+            return res
```

Comment:
I think we can inline this:
```suggestion
            return await req.json()
```